### PR TITLE
New UI

### PR DESCRIPTION
**Pull request by sergio-usma**
**### Project requirements**

The video with the UI example was watched again (it was seen while reading the Sneak Peek).
UI example

The console app behaved the same way as in the example.
An app.rb file was created that served as the console app entry point. It had methods that did the following:
- [X] Listed all books.
- [X] Listed all people.
- [X] Created a person (teacher or student, not a plain Person).
- [X] Created a book.
- [X] Created a rental.
- [X] Listed all rentals for a given person id.

In the main.rb file, the entry point was defined, and this method called "main" was invoked at the end of the file. This method did the following:
- [X] Presented the user with a list of options to perform.
- [X] Allowed users to choose an option.
- [X] If needed, asked for parameters for the option.
- [X] Had a way to quit the app.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+require_relative 'app'
+
+def main
+  app = App.new
+  app.run
+end
```
Comment (TadesseDev) [inline main.rb:6]: - [x] Dear @sergio-usma, Your app works fine till the point where I add a person and a book, I also confirm the rental is working fine. However, once I added those details, I could not see a list of books, the app kept crashing when I tried to list available books. Please make sure it's possible to see a list of books when a user selects option 1
Screen Shot 2023-09-10 at 12 30 00 AM

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi Team

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## Highlights
 - [x] You have written a clean code ✅
 - [x]  Most of the requirement has been meet. ✅
 - [x]  Adding a book, a person, and renting a book is working well ✅  
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb (hunk 1 above)
Comment (sergio-usma) [reply main.rb:6]: Ok bro! The bug was solved!!! Thank you very much for your review!!

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @sergio-usma  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Linter checks passing :heavy_check_mark: 
- Made changes as requested by the previous reviewer :+1: 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**